tko_hr_pt/__init__.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Module names should not identify the company name.
I suggest `l10n_pt_hr`, following what is done in other localizations.